packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -581,3 +579,32 @@ export async function afterTest(
     await compass.capturePage(screenshotPathName(test.fullTitle()));
   }
 }
+
+const SENSITIVE_ENV_VARS = [
+  'E2E_TESTS_ATLAS_PASSWORD',
+  'E2E_TESTS_ATLAS_IAM_ACCESS_KEY_ID',
+  'E2E_TESTS_ATLAS_IAM_SECRET_ACCESS_KEY',
+];
+
+function redact(value: string): string {
+  for (const field of SENSITIVE_ENV_VARS) {
+    if (process.env[field] === undefined) {
+      continue;
+    }
+
+    const quoted = `'${process.env[field] as string}'`;
+    // /regex/s would be ideal, but we'd have to escape the value to not be
+    // interpreted as a regex.
+    while (value.indexOf(quoted) !== -1) {
+      value = value.replace(quoted, "'<redacted>'");
```

Comment:
Maybe this could be 

```suggestion
      value = value.replace(quoted, "'$" + field + "'");
```

to be more meaningful if/when we add other sensitive env vars?

Reply:
Nice.